samples/go/nomdex/nomdex_find.go

```diff
@@ -18,42 +18,54 @@ import (
 	flag "github.com/juju/gnuflag"
 )
 
-var longHelp = `Find retrieves and prints objects that satisfy the 'query' argument.
+var longFindHelp = `'nomdex find' retrieves and prints objects that satisfy the 'query' argument.
 
-Indexes are built using the 'nomdex up' command. Once built, the indexes can be referenced
-in the 'query' arg to select objects matching certain criteria. For example, if there are
-objects in the database that contain a personId and a gender field, 'nomdex up' can scan all
-the objects in a given dataset and build an index on the specified field with the following
-commands:
-   nomdex up --by gender --in-path <dsSpec> --out-ds gender-index
-   nomdex up --by personId --in-path <dsSpec> --out-ds personId-index
+Indexes are built using the 'nomdex up' command. For information about building
+indexes, see: nomdex up -h
 
-Once these indexes are built, objects can be retrieved quickly and efficiently using the
-nomdex query language. For example, the followign query could be used to find all people
-with with a personId between 1 and 2000 and who are female:
-    nomdex find '(personId >= 0 and personId <= 2000) and gender = "female"
+Objects that have been indexed can be quickly found using the nomdex query
+language. For example, consider objects with the following type:
 
-The next command would retrieve all people objects that were either male or had an personId
-greater than 2000:
-    nomdex find 'gender = "male" or personId > 2000'
+struct Person {
+  name String,
+  geopos struct GeoPos {
+     latitude Number,
+     longitude Number,
+  }
+}
 
+Objects of this type can be indexed on the name, latitude and longitude fields
+with the following commands:
```

Comment:
Does it matter if these are in a List, Set, or Map?  You don't really say what people.value is below - does it need to be a Collection? Might be useful to specify what you are assuming in this example - e.g. "Consider objects of the following type in a Noms list".